controllers/userGroupController.js

```diff
@@ -1,20 +1,28 @@
+import { cyan, red } from 'colors';
+import { ConsoleLogger } from '../utils/logger';
 import { createUserGroupService, getUserGroupsService } from '../services/userGroupService';
 
 export default class GroupController {
     async createUserGroup(req, res) {
-        const createdUserGroup = await createUserGroupService(req.body);
-
-        if (createdUserGroup.errorCode === 409) {
-            return res.status(409).json(createdUserGroup);
+        try {
+            ConsoleLogger.info(`${cyan('Request')}: ${req.method} ${req.url}, ${cyan('Query')}: ${JSON.stringify(req.body)}`);
+            const createdUserGroup = await createUserGroupService(req.body);
+            res.json(createdUserGroup);
+        } catch (error) {
+            ConsoleLogger.error(`${cyan('Request')}: ${req.method} ${req.url}, ${red('Error')}: ${error.message}`);
+            res.status(error.status).json(error);
         }
-
-        res.json(createdUserGroup);
     }
 
     async getUserGroups(req, res) {
-        const { rows, count } = await getUserGroupsService(req.query);
-
-        res.json({ userGroups: rows, count });
+        try {
+            ConsoleLogger.info(`${cyan('Request')}: ${req.method} ${req.url}, ${cyan('Query')}: ${JSON.stringify(req.query)}`);
+            const { rows, count } = await getUserGroupsService(req.query);
+            res.json({ userGroups: rows, count });
+        } catch (error) {
+            ConsoleLogger.error(`${cyan('Request')}: ${req.method} ${req.url}, ${red('Error')}: ${error.message}`);

```

Comment:
`Every method in the controllers should log the errors which should include the following information:
  −method name;
  −arguments which have been passed to the method;
  −error message.`
  
  Information not according to the task